docs/openapi.yaml

```diff
@@ -0,0 +1,181 @@
+openapi: 3.1.0
+info:
+  title: ACED Submission
+  contact: {}
+  version: 0.0.1
+servers:
+- url: https://aced-idp.org/Bundle
+  description: ACED FHIR Bundle Implementation
+paths:
+  /Bundle:
```

Comment:
Would be very useful to add a fhir individual resource submission to the openapi too